### ISSUE TITLE
Re-encrypt store to new gpg key

### ISSUE THREAD

**Issue by wakatara** [gopasspw/gopass#1256]
In the classic `pass`, issuing `pass init <gpg_id>` decrypts the old store and then re-encrypts to the new key specified by the ID. However, it gopass behaviour with the `gopass init <gpg_id>` gives:

```
 λ ~ gopass init ABCDEFGH
[init] Initializing a new password store ...
[init] WARNING: Store is already initialized
Error: failed to initialize store: failed to init store '' at '/Users/daryl/.password-store': failed to initialize new sub store: Found already initialized store at /Users/daryl/.password-store.
...
```

Is there a mechanism to re-encrypt the *entire* store to a new gpg key? I've not been able to figure out how to do this from the gopass docs (I may just download classic `pass` and do it that way) but since gopass is compatible and this is an excellent convenience feature I was just wondering if this feature is missing or there is another way to do it (and happy to update the docs somewhere when you tell me... =] ).

**Comment by MartinAltmayer** [comment]
I did this by hand last week. Reencryption itself is simple:
```
gopass recipients add <keyid>
gopass recipients remove <keyid>
```
However, the Git history will still contain your passwords encrypted with the old key. To fix this, you can delete the .git-folder within .password-store and recreate it:
```
cd .password-store
rm -rf .git
git init
git add .public-keys .gitattributes .gpg-id *
git commit -m"Initial commit after changing GPG key"
```
Please make a backup of your store before using this procedure!

**Comment by wakatara** [comment]
@MartinAltmayer 

Now you've written it down, I'm a bit embarrassed I didn't think of that route. :-/. Thank you very much for the prompt response. I'll add it to my blog post on using gopass (but would also suggest it would be a really good thing to add to the docs as well -- happy to do that myself with a PR if you tell me where you feel it should go in the docs. I have to admit, because I kept thinking of the `pass init` command doing it, I did not think how to solve it in this more clever fashion. 
)

thanks for the help!
Daryl
